google/cloud/spanner/row_test.cc

```diff
@@ -284,6 +287,36 @@ TEST(Row, ValuesAccessorRvalue) {
   EXPECT_EQ(kData, *v);
 }
 
+TEST(Row, ValuesConst) {
+  auto const row = MakeRow("foo", 12345, "bar");
+  auto values = row.values();
+  EXPECT_EQ(3, values.size());
+  EXPECT_EQ(Value("foo"), values[0]);
+  EXPECT_EQ(Value(12345), values[1]);
+  EXPECT_EQ(Value("bar"), values[2]);
+}
+
+TEST(Row, ValuesMove) {
+  auto row = MakeRow("foo", 12345, "bar");
+  auto values = std::move(row).values();
+  EXPECT_EQ(3, values.size());
+  EXPECT_EQ(Value("foo"), values[0]);
+  EXPECT_EQ(Value(12345), values[1]);
+  EXPECT_EQ(Value("bar"), values[2]);
+}
+
+TEST(Row, PrinTo) {
+  auto row = MakeRow(1234, "foo", 2345, "bar", "baz");
+  std::ostringstream os;
+  PrintTo(row, &os);
+  auto actual = os.str();
+  EXPECT_THAT(actual, HasSubstr("1234"));
+  EXPECT_THAT(actual, HasSubstr("2345"));
```

Comment:
Is the non-column ordering of expectations supposed to indicate anything?

Reply:
Nope, fixed.